### ISSUE TITLE
Error in parse_git_repo(repo) : could not find function "setNames" Calls: <Anonymous> -> lapply -> FUN -> parse_git_repo

### ISSUE THREAD

**Issue by englianhu** [hadley/devtools#772]
```{r}
> require(devtools)
> install_github("cscheid/rgithub")
Downloading github repo cscheid/rgithub@master
Installing github
Installing dependencies for github:
Rook
trying URL 'http://cran.rstudio.com/bin/windows/contrib/3.2/Rook_1.1-1.zip'
Content type 'application/zip' length 281971 bytes (275 KB)
downloaded 275 KB

package 慠ook?successfully unpacked and MD5 sums checked

The downloaded binary packages are in
	C:\Users\User\AppData\Local\Temp\Rtmp6ncqzH\downloaded_packages
"C:/PROGRA~1/R/R-32~1.0RC/bin/x64/R" --vanilla CMD INSTALL  \
  "C:/Users/User/AppData/Local/Temp/Rtmp6ncqzH/devtools99835a94b12/cscheid-rgithub-ead1c0d"  \
  --library="C:/Program Files/R/R-3.2.0rc/library" --install-tests 

Error in parse_git_repo(repo) : could not find function "setNames"
Calls: <Anonymous> -> lapply -> FUN -> parse_git_repo
Execution halted
Error: Command failed (1)
> require(devtools)
> install_github("brian-bot/rGithubClient")
Downloading github repo brian-bot/rGithubClient@master
Installing rGithubClient
"C:/PROGRA~1/R/R-32~1.0RC/bin/x64/R" --vanilla CMD INSTALL  \
  "C:/Users/User/AppData/Local/Temp/Rtmp6ncqzH/devtools99838211e27/brian-bot-rGithubClient-ca29bba"  \
  --library="C:/Program Files/R/R-3.2.0rc/library" --install-tests 

Error in parse_git_repo(repo) : could not find function "setNames"
Calls: <Anonymous> -> lapply -> FUN -> parse_git_repo
Execution halted
Error: Command failed (1)```

**Comment by wch** [comment]
I think this would be fixed by #734.

**Comment by englianhu** [comment]
```{r}> install_github("cscheid/rgithub")
Downloading github repo cscheid/rgithub@master
Installing github
"C:/PROGRA~1/R/R-32~1.0/bin/x64/R" --vanilla CMD INSTALL  \
  "C:/Users/Scibrokes/AppData/Local/Temp/RtmpyYylPq/devtools254c32b2430a/cscheid-rgithub-ead1c0d"  \
  --library="C:/Users/Scibrokes/Documents/R/win-library/3.2" --install-tests 

Error in parse_git_repo(repo) : could not find function "setNames"
Calls: <Anonymous> -> lapply -> FUN -> parse_git_repo
Execution halted
Error: Command failed (1)```
